### PR TITLE
adds multisig/aggregateSigningShares RPC

### PR DESCRIPTION
**Pull request by hughy**
## Summary

takes a publicKeyPackage, an unsignedTransaction, a signingPackage for that transaction, and a list of signatureShares for that signingPackage

uses UnsignedTransaction.signFrost to aggregate the signatureShares, sign the transaction, and produce a signed transaction

returns a signed transaction serialized as hex

## Testing Plan

RPC integration test will be added in a followup PR

## Documentation

Does this change require any updates to the Iron Fish Docs (ex. [the RPC API
Reference](https://ironfish.network/docs/onboarding/rpc/chain))? If yes, link a
related documentation pull request for the website.

```
[ ] Yes
```

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish/src/rpc/clients/client.ts [hunk 1]
```diff
@@ -821,6 +823,15 @@ export abstract class RpcClient {
     },
   }
   multisig = {
+    aggregateSignatureShares: (
```
Comment (jowparks) [inline ironfish/src/rpc/clients/client.ts:826]: We should either rename this to match `signFrost` or rename `signFrost` to match this.

**Review by jowparks — APPROVED** [review]
Looks good, we just need to rename this (or signFrost) and in a subsequent PR write integration test like you suggested offline.

File: ironfish/src/rpc/clients/client.ts [hunk 2]
```diff
@@ -821,6 +823,15 @@ export abstract class RpcClient {
...
+      params: AggregateSigningSharesRequest,
+    ): Promise<RpcResponseEnded<AggregateSigningSharesResponse>> => {
+      return this.request<AggregateSigningSharesResponse>(
+        `${ApiNamespace.multisig}/aggregateSignatureShares`,
```
Comment (hughy) [inline ironfish/src/rpc/clients/client.ts:830]: typo: `s/Signature/Signing/`

File: ironfish/src/rpc/clients/client.ts (hunk 1 above)
Comment (hughy) [reply ironfish/src/rpc/clients/client.ts]: Merging as is for now, we'll settle on naming in a followup

**Pull request merged**